### ISSUE TITLE
Wrong answer confirmation with Anki ⁨2.1.45 (355e66e8)⁩

### ISSUE THREAD

**Issue by rwmpelstilzchen** [Quip13/No-Distractions-Full-Screen#34]
Hi,

Thanks for NDFS! It makes my Anki reviews much more pleasant 😊

I’ve just installed the newest version of Anki, and NDFS’s answer confirmation doesn’t work well with it: every time a question is shown (including the first one in the review session, before answering any question), NDFS displays a confirmation for the *current* card, including its next review time. 

I’m not sure if it is a bug in the new version of Anki or a compatibility issue.

For now I disabled the confirmation option as a workaround.

Júda

**Comment by ccz-2** [comment]
Seems to be an issue with the new scheduler. Can you try the v2 scheduler
and see if it works?

Unfortunately I don't have time rn to fix the issue. I'll get to it in a
few weeks

**Comment by rwmpelstilzchen** [comment]
This occurs with v2. I haven’t switch to v3 yet, because I use desktop Anki in tandem with AnkiDroid, which doesn’t have v3 support yet.

**Comment by ccz-2** [comment]
Thanks for the info. I'll have to do a deeper dive into this when I get some free time

**Comment by rwmpelstilzchen** [comment]
Thanks! 🙂

**Comment by felipemldias** [comment]
Hey @rwmpelstilzchen, did you find any other workaround for this particular issue by any chance?

**Comment by rwmpelstilzchen** [comment]
No, @felipemldias, sorry. I have the answer bar hidden (View → HD Full Screen → Answer Button Visibility → Hidden) and use the keyboard for reviewing, so it doesn’t bother me…
